apps/admin-gui/src/app/vos/pages/group-detail-page/group-applications/group-applications.component.html

```diff
@@ -44,12 +36,26 @@ <h1 class="page-subtitle">{{'VO_DETAIL.APPLICATION.TITLE' | translate}}</h1>
     [placeholder]="'VO_DETAIL.APPLICATION.FILTER'"></perun-web-apps-debounce-filter>
   <perun-web-apps-date-range (datePicker)="select()" [endDate]="endDate" [startDate]="startDate">
   </perun-web-apps-date-range>
+  <div>
+    <button
+      [matTooltipDisabled]="!showAllDetails"
+      matTooltip="{{'VO_DETAIL.APPLICATION.COLUMNS_TOOLTIP' | translate}}"
```

Comment:
Toolpit on disabled elements does not show up on certain browsers (e.g. chrome). Put the tooltip on the div that is parent of the button.